src/test/java/com/intellias/basicsandbox/controller/ItemControllerMvcTests.java

```diff
@@ -14,19 +14,22 @@
 import com.intellias.basicsandbox.persistence.entity.ItemEntity;
 import com.intellias.basicsandbox.service.ItemService;
 import com.intellias.basicsandbox.service.exception.ItemNotFoundException;
+
 import java.nio.charset.StandardCharsets;
 import java.util.Locale;
 import java.util.UUID;
+
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.MediaType;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 
-@WebMvcTest(ItemController.class)
+@WebMvcTest(value = ItemController.class, excludeAutoConfiguration = SecurityAutoConfiguration.class)
```

Comment:
the idea to use 2 test types with or without security otherwise you need to enable/disable security for every controller test. Consider having the interfaces that enable/disable security and implement the appropriate